lib/embed/gpi_embed.c

```diff
@@ -50,6 +50,22 @@ static char *argv[] = { progname };
 
 static PyObject *pEventFn = NULL;
 
+/**
+ * @name    Print the traceback
+ * @brief   Print traceback to stderr
+ *
+ * GILState before calling: N/A
+ *
+ * GILState after calling: N/A
+ *
+ * Print traceback and flush stderr if required (on Windows)
+ */
+void print_traceback(void) {
+  PyErr_Print();
+#if defined(_WIN32)
+  PyRun_SimpleString("import sys; sys.stderr.flush()");
```

Comment:
@LinuxFanatic777 What lines should I add in some example from repo to reproduce the problem? I'm just curious, because I've never have any problem =)